rusthelloserverless/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+edition = "2021"
+name = "cloud"
+version = "0.0.0"
+
+[dev-dependencies]
+scylla = {version = "0.8.0", features = ["ssl", "cloud"]}
+tokio = {version = "1.1.0", features = ["full"]}
```

Comment:
Let's use the newest driver available: `0.9.0`, as well as `tokio` version matching the one used by the driver `1.27`: